Src/Plugins/Tests.Security/SEC101.SecurePlaintextSecretsTests.cs

```diff
@@ -19,5 +19,9 @@ public SecurePlaintextSecretsTests(ITestOutputHelper outputHelper) : base(output
         [Fact]
         public void SecurePlaintextSecrets_EndToEndFunctionalTests()
             => RunAllTests();
+
+        [Fact]
+        public void SecurePlaintextSecrets_ValidateOutputFiles()
+            => ValidateOutputFiles();
```

Comment:
>ValidateOutputFiles [](http://example.com/codeflow?start=15&length=19)

This new method will validate our expected sarif files and search for issues, for example, an empty sarif.

Reply:
removed the test since i added as a following verification after the RunAllTests method.